src/components/Table/GridTable.tsx

```diff
@@ -370,19 +326,21 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
       const length = rows.length;
       rows.forEach((row, i) => {
         if (row.kind === "header") {
+          nestedCards && nestedCards.maybeOpenCard(row);
```

Comment:
This is to make the header look like a card? I kinda thought the 1st set of mocks had "doesn't look like a card" headers, but I guess now they do? That's cool, just checking.

Reply:
It shouldn't, looking at the screenshot, but I think that is because I removed the borderRadius. 

I will see if I can remove this and still get the style that I want.

- [ ] Verify if this change is needed.

Reply:
Ah okay, cool, that makes sense.